packages/lesswrong/lib/collections/moderatorActions/schema.ts

```diff
@@ -10,8 +16,16 @@ export const MOVED_POST_TO_DRAFT = 'movedPostToDraft';
 export const SENT_MODERATOR_MESSAGE = 'sentModeratorMessage';
 export const MANUAL_FLAG_ALERT = 'manualFlag';
 
+export const rateLimits = [RATE_LIMIT_ONE_PER_DAY, RATE_LIMIT_ONE_PER_THREE_DAYS, RATE_LIMIT_ONE_PER_WEEK, RATE_LIMIT_ONE_PER_FORTNIGHT, RATE_LIMIT_ONE_PER_MONTH] as const
+
+export type RateLimitType = typeof rateLimits[number]
```

Comment:
As mentioned [here](https://github.com/ForumMagnum/ForumMagnum/pull/6057/files#r1022141857), probably worth making a `TupleSet` out of `rateLimits`, and then pulling `RateLimitType` out of it using the accompanying type helper `UnionOf` (`UnionOf<typeof rateLimits>`)